docker/README.md

```diff
@@ -1,6 +1,6 @@
 # Harpin AI Toolkit - Docker Image
 
-Download our harpin AI Toolkit Docker image and try out our identity resolution algorithm on your data. Run the Docker image anywhere you have access to the data and a Docker runtime, in your cloud or on your own workstation. The harpin AI shell will walk you through the setup and configuration process and have the resolution process running in Docker in no time at all.
+Download our harpin AI Toolkit Docker image and try out our identity resolution algorithm and data quality analysis on your data. Run the Docker image anywhere you have access to the data and a Docker runtime, in your cloud or on your own workstation. The harpin AI shell will walk you through the setup and configuration process and have the resolution process running in Docker in no time at all.
 
 The image supports executing identity resolution on up to 5 million records as part of the trial. For handling more than 5 million records, please contact [Harpin AI](https://harpin.ai/demo/) for additional options.
```

Comment:
I assume the 5M row limit applies to data quality analysis as well?  If so, should we remove "identity resolution" from this line and just reference harpin AI in general?